README.md

```diff
@@ -81,6 +81,30 @@ end
 # will produce => "2011HOUSE00001", "2011HOUSE00002"...
 ```
 
+It's possible to pass :scope_by option as a simple method, Proc.new or lambda
+
+Ex:
+```ruby
+# :manufacturer should be a Car's instance method(like an ActiveRecord column)
+class Car < ActiveRecord::Base
+    protokoll :code, :scope_by => :manufacturer
```

Comment:
I feel we should have a single name for this. `scope_by` should accept a lambda or a method name.
this way it simplifies the API.

thoughts?